media/css/firefox/mobile-download-desktop.less

```diff
@@ -0,0 +1,211 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+@import "../sandstone/fira-fonts.less";
```

Comment:
Including the Fira fonts as well as Open Sans in the CSS means some browsers (*cough* IE) will download both sets of fonts. I think we should either be using Sandstone (with Open Sans) or Pebbles (with Fira), but we should generally be avoiding mixing CSS that uses both font families.